### PR TITLE
Docs: fixed table of contents aligment

### PR DESCRIPTION
**Pull request by qwerty541**

### CODE REVIEW

**Comment by rickstaa** [comment]
@qwerty541 I agree that there might be something wrong with the sections. The TOC is, however, automatically created by the https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one extension. As a result, if you think the alignment is not correct, you should change the heading levels (e.g. `### All Demos` -> `# All Demos`.

File: readme.md [hunk 1]
```diff
@@ -106,14 +106,14 @@ Please visit [this link](https://give.do/fundraisers/stand-beside-the-victims-of
...
     - [Demo](#demo-2)
-    - [All Demos](#all-demos)
-    - [Quick Tip (Align The Repo Cards)](#quick-tip-align-the-repo-cards)
-  - [Deploy on your own](#deploy-on-your-own)
-    - [On Vercel](#on-vercel)
-    - [On other platforms](#on-other-platforms)
-    - [Disable rate limit protections](#disable-rate-limit-protections)
-    - [Keep your fork up to date](#keep-your-fork-up-to-date)
-  - [:sparkling\_heart: Support the project](#sparkling_heart-support-the-project)
+- [All Demos](#all-demos)
```
Comment (rickstaa) [inline readme.md:109]: This should be changed on the heading side, as this is automatically generated using the https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one extension.

**Review by rickstaa — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (qwerty541) [reply readme.md:121]: I have updated heading levels and regenerated TOC using `yzhang.markdown-all-in-one` extension. Please take a look.

**Review by rickstaa — APPROVED** [review]

**Comment by rickstaa** [comment]
I decreased the TOC depth to `"markdown.extension.toc.levels": "1..3"` so that it is consistent with the old behaviour. I think now we are good to merge 👍🏻.

**Pull request merged**